hub/apps/windows-app-sdk/system-requirements.md

```diff
@@ -39,16 +39,13 @@ The [Windows App SDK](index.md) is a set of developer tools that represent the n
 
 The Windows App SDK has these system requirements:
 
-- Windows 10, version 1809 (build 17763) and later.
+- Windows 10, version 1809 (build 17763) or later
 
-- Visual Studio 2019 version 16.9 and later with the following workloads and components:
-  - **Universal Windows Platform development**
-  - **.NET Desktop Development** (needed even if you're only building C++ Win32 apps)
-  - **Desktop development with C++** (needed even if you're only building .NET apps)
+- Visual Studio 2019 version 16.9 or later with the required components (see [Required workloads and components](https://docs.microsoft.com/windows/apps/windows-app-sdk/set-up-your-development-environment?tabs=vs-2022-17-1-a%2Cvs-2022-17-1-b#required-workloads-and-components))
 
-- Windows SDK version 2004 (build 19041) and later. This is installed with Visual Studio 2019 and Visual Studio 2022 by default.
+- Windows SDK version 2004 (build 19041) or later, as included with Visual Studio 2019 and Visual Studio 2022 by default
 
-- Building .NET apps also requires .NET 6 or later (see [Download .NET](https://dotnet.microsoft.com/download) and [.NET 5 will reach End of Support on May 10, 2022](https://devblogs.microsoft.com/dotnet/dotnet-5-end-of-support-update/)).
+- Building .NET apps also requires .NET 6 or later (see [Download .NET](https://dotnet.microsoft.com/download) and [.NET 5 will reach End of Support on May 10, 2022](https://devblogs.microsoft.com/dotnet/dotnet-5-end-of-support-update/))
```

Comment:
```suggestion
- Building .NET apps also requires .NET 6 or later (see [Download .NET](https://dotnet.microsoft.com/download)
```

removing the .net 5 mention since it's not as relevant anymore

Reply:
Good eye @asklar !